### ISSUE TITLE
kendo.ui 

### ISSUE THREAD

**Issue by hosain-cse2** [tjwebb/react-kendo#5]
Hi,

I am trying to use the react-kendo in my react project. I want to use the context menu.
## Following is my code,

import React from 'react';
var k = React.Kendo = require('react-kendo');

var contextMenuOptions = {
    target: "#target",
    alignToAnchor: true
};

export default class ContextMenu extends React.Component{

```
constructor(){
    super();
}

render(){

    return  (
        <div>
            <h1 id="target">Right click here...</h1>
            <k.kendoContextMenu options={contextMenuOptions}></k.kendoContextMenu>
        </div>
    );
}
```
## }

But the problem is when I load the page i am getting the following error,

```
                                Error: kendo.ui not found
```

I am also using the webpack. Can you please tell me how can I solve this problem? 
Do I need to install any other packages?

**Comment by mikemorton** [comment]
Can you show us the content of your page, including all the scripts you're including?

Are you sure you're including the KendoUI script in your page?

**Comment by denizdogan** [comment]
Could this be related to #7?

**Comment by dlucidone** [comment]
I'm getting the same error. Is there any way out Thanks!

**Comment by shriramdevanathan** [comment]
@hosain-cse2 Were you able to resolve the issue ? ?

**Comment by hosain-cse2** [comment]
No. I actually moved into the Material-UI framework. So I did not try it again.

**Comment by ravindra-tm** [comment]
I included the kendo js script in main.js and using it
let say you have a component in render as follow - 
`<div id="TableGrid"></div>`

Now define the grid in class definition as -
`$("#TableGrid").kendoGrid({
                    columns: [],
                    editable: "inline",
                    sortable: true,
                    reorderable: true,
                    resizable: true,
                    filterable: true,
                    columnMenu: true,
                    pageable: true,
                    dataSource: ds1,
                });`

If need help ping me!

**Comment by nwhitmont** [comment]
Same issue:
```
~/<project>/node_modules/react-kendo/lib/index.js:8
  throw new Error('kendo.ui not found');
        ^
Error: kendo.ui not found
```

Are there some instructions how to resolve this issue?
